setup.py

```diff
@@ -55,9 +55,21 @@ def get_version():
         for line in lines:
             if line.startswith('Version:'):
                 return line[8:].strip()
-    if os.path.exists("./tools"):
-        print("Running tools/get_version.sh")
-        version = subprocess.getoutput("bash -c 'cd tools; ./get_version.sh'")
+    if os.path.exists("./include/vrv"):
+        with open("./include/vrv/vrvdef.h") as header_file:
+            defines = {}
+            for line in header_file.readlines():
+                if line.startswith("#define"):
+                    line.rstrip()
+                    definition = line[8:].split()
+                    try:
+                        defines[definition[0]] = definition[1]
```

Comment:
This adds all the definitions to the dictionary, which seems unnecssary given that we're only looking for three or four?

Reply:
Now skipping when `vrv_cast` is reached. Maybe we want to use more defines from there later on.